user/database-setup.md

```diff
@@ -64,47 +64,33 @@ test:
   encoding: utf8
 ```
 
-You might have to create the `myapp_test` database first. Run this as part of your build script:
+You might have to create the `myapp_test` database first, for example in
+the `before_install` step in `.travis.yml`:
 
 ```yaml
-# .travis.yml
-before_script:
-  - mysql -e 'create database myapp_test;'
+before_install:
+  - mysql -e 'CREATE DATABASE myapp_test;'
 ```
 
 ### Note on `test` database
 
-In older versions of MySQL, Ubuntu package provided the `test` database by default.
-This is no longer the case as of version 5.5.37 due to security concerns
-(See [change log](http://changelogs.ubuntu.com/changelogs/pool/main/m/mysql-5.5/mysql-5.5_5.5.47-0ubuntu0.12.04.1/changelog)).
+In older versions of MySQL, the Ubuntu package provided the `test` database by
+default.  This is no longer the case as of version 5.5.37 due to security
+concerns (See [change
+log](http://changelogs.ubuntu.com/changelogs/pool/main/m/mysql-5.5/mysql-5.5_5.5.47-0ubuntu0.12.04.1/changelog)).
```

Comment:
This section is historical data, and can disappear (oldest version on Precise is 5.5.41).